js/main.js

```diff
@@ -0,0 +1,127 @@
+var currentGyro = 0;
+var offsetGyro = 0;
+
+$(document).ready(function() {
+
+    // sets a function that will be called when the websocket connects/disconnects
+    NetworkTables.addWsConnectionListener(onNetworkTablesConnection, true);
+
+    // sets a function that will be called  when the robot connects/disconnects
+    NetworkTables.addRobotConnectionListener(onRobotConnection, true);
+
+
+    // sets a function that will be called when any NetworkTables key/value changes
+    NetworkTables.addGlobalListener(onValueChanged, true);
+
+    // hook up our SendableChoosers to combo boxes
+    attachSelectToSendableChooser("#auto-selector", "/SmartDashboard/Autonomous Mode");
+
+    $("#mainform").submit(function(e) {
+        e.preventDefault();
+        remove_form();
+    });
+
+});
+
+function resetGyro() {
+
+    offsetGyro = currentGyro;
+    rotateCompass(currentGyro + Math.PI);
+}
+
+
+function onNetworkTablesConnection(connected) {
+    if (connected) {
+        null;
+    } else {
+        null;
```

Comment:
Expected an assignment or function call and instead saw an expression.

Reply:
Function required to remove checklist revealing the camera